docs/Versioning.md

```diff
@@ -140,3 +140,12 @@ ISSUE: what's our policy when a model takes a dependency on new `IR_VERSION` cha
 ### Accuracy or performance changes
 
 Assuming that there are no breaking changes to the signature of the model's graph or any operator dependencies, the shape and contents of the graph can change freely provided there are no semantic changes to the model. However, changes to the shape and contents of the graph can impact model accuracy and/or model performance.
+
+## Released Versions
+
+ONNX version|File format version|Operator set version ai.onnx|Operator set version ai.onnx.ml
+------------|-------------------|----------------------------|-------------------------------
+1.0|3|1|1
```

Comment:
Here do we only cover the major version or we also want to cover the minor version? For example, in 1.1.2, opset version is 6.

Reply:
@houseroad Certainly - I let this PR stall. Updating...